config/charts/knative-operator/templates/operator.yaml

```diff
@@ -114,14 +130,23 @@ spec:
           value: knative.dev/operator
         - name: KUBERNETES_MIN_VERSION
           value: "{{ .Values.knative_operator.kubernetes_min_version }}"
+        {{- if and .Values.knative_operator.operator_webhook.securityContext }}
+        securityContext:
```

Comment:
There is also another securityContext on the pod level

Reply:
but the yaml does not have it be default, if .Values.knative_operator.operator_webhook.securityContext is not available, do not add it.